kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/MarshallingFormat.java

```diff
@@ -53,6 +57,20 @@ public static MarshallingFormat fromId(int id) {
         }
     }
 
+    public static boolean isStrictType(String type) {
+        int idx;
+        if ((idx = type.indexOf(';')) < 0 || (idx + 1) == type.length()) {
+            return false;
+        }
+        // we map parameter=1, pararmeter=2,.... into a map
+        Map<String, String> parameters = Arrays.asList(type.substring(idx + 1).split(","))
+                                               .stream()
+                                               .filter(e -> e.split("=").length > 1) // remove bad parameters
```

Comment:
I would even say that it has to equal to 2. But that just a cosmetic point of view.